contrib/updates/docker-compose.yml

```diff
@@ -72,7 +76,62 @@ services:
     networks:
       localnet:
         ipv4_address: 192.168.10.5
+  terradnode4:
+    container_name: terradnode4
+    platform: linux/amd64
+    image: "classic-terra/terrad-upgrade-env"
+    environment:
+      - ID=4
+      - LOG=terrad.log
+      - DAEMON_HOME=/terrad/node4/terrad
+      - DAEMON_NAME=terrad
+      - DAEMON_RESTART_AFTER_UPGRADE=true
+    ports:
+      - "9094:9090"
+      - "26665-26666:26656-26657"
+    volumes:
+      - ../../build:/terrad:Z
+    networks:
+      localnet:
+        ipv4_address: 192.168.10.6
```

Comment:
We should be able to source these IPs via the etcd store in the container as long as we do not target default networks. For now just leave it as-is, but its really something I would like to fix as long as we maintain these assets, as it would let us scale  higher then the "hardcoded" number of "docker-compose services"

Reply:
this will just be temporary fix until I get the terra-operator testnet feature ready to integrate